designer/client/src/components/tips/error/NodeErrorLink.tsx

```diff
@@ -37,7 +37,8 @@ export const NodeErrorLink = (props: { onClick: MouseEventHandler<HTMLAnchorElem
         </span>
     ) : (
         <Link className={styles} to={`?nodeId=${nodeId}`} onClick={onClick}>
-            {nodeId}
+            {/* blank values don't render as links so this is a workaround */}
+            {nodeId.trim() === "" ? "blank" : nodeId}
```

Comment:
What happens when I click on this blank link, is it an open node edit view?

Reply:
Yes, the router link component still gets the same `to` prop - this just changes what value the wrapped <a> tag has.

Reply:
The routing somehow works for blank nodes, the `?nodeId= ` prop  still contains the blank string so if you have 2 nodes with ids: " " and "   " it will still route correctly. The "blank" string is just for the link to be rendered  :smile: When they are repeated they just display the same value like this:
![image](https://github.com/TouK/nussknacker/assets/92804917/63c16a09-7695-43f1-a40d-4993c79f5e76) Other solution would be to put quotaton marks around the whitespaces.

Reply:
If it works, let's leave it ;).